assets/js/script.js

```diff
@@ -141,7 +141,7 @@ function isWinner(a, b) {
         setTimeout(() => {
             document.getElementById("result").innerHTML = a == 3 ? "Player 1 Win" : "Player 2 Win";
         }, 0);
-      
+      document.getElementById("result").style.padding='10px';
```

Comment:
Add padding in css

Reply:
if i am adding padding in css to result then it is showing like this:
![image](https://github.com/malivinayak/Tic-Tac-Toe/assets/112965155/b4f86322-9d86-47d9-97df-c3ca3a9d4a2b)
i think the js approach is appropriate because result is only visible when the game is over.

Reply:
ohk keep as it is